drivers/volume/portworx/portworx.go

```diff
@@ -330,19 +352,29 @@ func (d *portworx) StopDriver(n node.Node) error {
 	return d.schedOps.DisableOnNode(n)
 }
 
-func (d *portworx) WaitStart(n node.Node) error {
-	var err error
-	// Wait for Portworx to become usable.
+// Wait for Portworx to be up
+func (d *portworx) WaitStart() error {
 	t := func() (interface{}, error) {
-		clusterManager := d.getClusterManager()
-		if status, _ := clusterManager.NodeStatus(); status != api.Status_STATUS_OK {
+		if status, _ := d.getClusterManager().NodeStatus(); status != api.Status_STATUS_OK {
```

Comment:
I don't understand the use of this function if we have WaitForNode. This function is only going to check the status of the node which is currently the endpoint.

If the intention of this function is to just check Cluster status then use
clusterManager.Enumerate() which returns api.Cluster object which has cluster status.

Reply:
done in incremental